### ISSUE TITLE
Image Upload

### ISSUE THREAD

**Issue by manavkataria** [manavkataria/behavioral_cloning_car#1]
## Raw Data Histogram: Asymmetric and Unbalanced
![Histogram: Asymmetric and Unbalanced](https://cloud.githubusercontent.com/assets/2206789/22631651/24bdcd60-ebc6-11e6-98f6-46f0cc1e926e.png)

## Horizontally Flipped Data Histogram: Symmetric But Unbalanced
![Histogram: Symmetric But Unbalanced](https://cloud.githubusercontent.com/assets/2206789/22631698/e0e85e38-ebc6-11e6-94cb-05f7739f188d.png)

## Fully Processed with Histogram Equalization: Symmetric and Balanced
![Histogram Equalization: Symmetric and Balanced](https://cloud.githubusercontent.com/assets/2206789/22631705/f71f9f5e-ebc6-11e6-9315-c4fc5f3048ab.png)

**Comment by manavkataria** [comment]
## Loss Function 5 Epochs
![Loss Function 5 Epochs](https://cloud.githubusercontent.com/assets/2206789/22679803/a52b2972-ecb9-11e6-83c1-c8b1eb066999.png)

## Predictions for 120 Frames
![Predictions for 120 Frames](https://cloud.githubusercontent.com/assets/2206789/22679856/e07909f4-ecb9-11e6-9840-46e91d3b68ca.png)

---
## Loss Function 30 Epochs
![Loss Function 30 Epochs](https://cloud.githubusercontent.com/assets/2206789/22679857/e079a576-ecb9-11e6-9e0a-f601946d25aa.png)

## Predictions for 12 Frames
![Predictions for 12 Frames](https://cloud.githubusercontent.com/assets/2206789/22707512/da290d60-ed27-11e6-8b18-81d697aa34c7.png)

**Comment by manavkataria** [comment]
## Youtube Video URL
![Youtube Video URL](https://cloud.githubusercontent.com/assets/2206789/22684159/082e3fb2-ecd0-11e6-9365-f458612d7d0d.png)

**Comment by manavkataria** [comment]
## Youtube Video 
[![Youtube Video](https://cloud.githubusercontent.com/assets/2206789/22684201/316bb47c-ecd0-11e6-92c7-66eb5790d286.jpg)](https://goo.gl/zhD2jV)

**Comment by manavkataria** [comment]
## Deep Learning Neural Network Model 
![model](https://cloud.githubusercontent.com/assets/2206789/22705301/dcde791c-ed1f-11e6-9354-3fd2d59aeb80.jpg)

**Comment by manavkataria** [comment]
### Recovery: Extreme Left of Lane
![center_2017_01_16_18_49_00_738](https://cloud.githubusercontent.com/assets/2206789/22707096/80e205fa-ed26-11e6-8f2d-114353c31d54.jpg)

### Drive Straight: Center of Lane
![center_2017_01_16_18_49_02_100](https://cloud.githubusercontent.com/assets/2206789/22707103/885d4f4c-ed26-11e6-8525-5d1a78465959.jpg)

### Recovery: Extreme Right of Lane
![center_2017_01_16_18_49_04_959](https://cloud.githubusercontent.com/assets/2206789/22707162/b5e5e5e6-ed26-11e6-829c-f119ac7ceafd.jpg)
